.github/CODEOWNERS

```diff
@@ -1,14 +1,22 @@
 # Defaults
 * @nomad-xyz/engineering
 
-# Core packages 
-packages/contracts-core/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/contracts-bridge/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/deploy/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/multi-provider/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/contracts-router/ @nomad-xyz/experience @nomad-xyz/protocol
-
-# SDK Packages 
-packages/sdk/ @nomad-xyz/experience @nomad-xyz/protocol 
-packages/sdk-govern/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/sdk-bridge/ @nomad-xyz/experience @nomad-xyz/protocol
+# Contracts packages
```

Comment:
`Contracts packages`  are pretty obvious / uncontroversial - should be @nomad-xyz/protocol